octree/basic_utils_test.go

```diff
@@ -93,8 +94,9 @@ func TestSplitIntoOctants(t *testing.T) {
 		test.That(t, len(filledLeaves), test.ShouldEqual, 1)
 		test.That(t, len(emptyLeaves), test.ShouldEqual, 7)
 		test.That(t, len(internalLeaves), test.ShouldEqual, 0)
-		test.That(t, filledLeaves[0].node.point, test.ShouldResemble, pc.PointAndData{P: p, D: d})
-		test.That(t, filledLeaves[0].checkPointPlacement(p), test.ShouldBeTrue)
+		test.That(t, filledLeaves[0].node.point, test.ShouldResemble,
+			pc.PointAndData{P: pointsAndData[0].P, D: pointsAndData[0].D})
+		test.That(t, filledLeaves[0].checkPointPlacement(pointsAndData[0].P), test.ShouldBeTrue)
```

Comment:
[opt] I'd use a for loop here (same as mentioned above)

Reply:
Done!